go/tools/builders/stdliblist.go

```diff
@@ -158,19 +190,28 @@ func stdliblist(args []string) error {
 		return err
 	}
 
+	cloneBase, cleanup, err := goenv.workDir()
+	if err != nil {
+		return err
+	}
+	defer func() { cleanup() }()
+
+	newGoRoot := filepath.Join(cloneBase, "external/go_sdk")
```

Comment:
now the clone logic is simpler -- just copy from abs(A) to abs(B), we just need to hardcode `external/go_sdk` here. Not sure if it's safe to assume so